### PR TITLE
sentiment-analysis multiple gpus

### PR DESCRIPTION
**Pull request by xiaotinghe**

### CODE REVIEW

File: chapter_natural-language-processing/sentiment-analysis-cnn.md [hunk 1]
```diff
@@ -322,20 +324,38 @@ def eval_model(features, labels):
...
+        self.y = y
+
+    def __getitem__(self, index):
+        return self.x[index], self.y[index]
+
+    def __len__(self):
+        return len(self.x)
+train_loader = DataLoader(MyDataset(train_features, train_labels), batch_size=batch_size, shuffle=True)
+test_loader = DataLoader(MyDataset(test_features, test_labels), batch_size=batch_size, shuffle=False)
+```
```
Comment (astonzhang) [inline chapter_natural-language-processing/sentiment-analysis-cnn.md]: Use gdata.ArrayDataset instead.
Example:
http://zh.gluon.ai/chapter_deep-learning-basics/linear-regression-gluon.html

File: chapter_natural-language-processing/sentiment-analysis-cnn.md [hunk 2]
```diff
@@ -304,16 +306,16 @@ loss = gloss.SoftmaxCrossEntropyLoss()
 
 在实验中，我们使用准确率作为评价模型的指标。
 
-```{.python .input  n=12}
+```{.python .input  n=95}
 def eval_model(features, labels):
```
Comment (astonzhang) [inline chapter_natural-language-processing/sentiment-analysis-cnn.md]: Let's use gb.evaluate_accuracy(). This function will be invoked by gb.train. Thus, just use gb.train() like in the following example:
http://zh.gluon.ai/chapter_computer-vision/fine-tuning.html

The definitions of gb.train() is available at:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L371

If it's impossible to simply invoke gb.train(), just change the function name to evaluate_accuracy() and make it be able to evaluate on multi-gpu, example:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L106

File: chapter_natural-language-processing/sentiment-analysis-cnn.md [hunk 3]
```diff
@@ -322,20 +324,38 @@ def eval_model(features, labels):
...
+    def __getitem__(self, index):
+        return self.x[index], self.y[index]
+
+    def __len__(self):
+        return len(self.x)
+train_loader = DataLoader(MyDataset(train_features, train_labels), batch_size=batch_size, shuffle=True)
+test_loader = DataLoader(MyDataset(test_features, test_labels), batch_size=batch_size, shuffle=False)
+```
+
 下面开始训练模型。
```
Comment (astonzhang) [inline chapter_natural-language-processing/sentiment-analysis-cnn.md]: Just use gb.train() like in the following example:
http://zh.gluon.ai/chapter_computer-vision/fine-tuning.html

The definitions of gb.train() is available at:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L371

File: chapter_natural-language-processing/sentiment-analysis.md [hunk 4]
```diff
@@ -15,11 +15,12 @@ import collections
 import gluonbook as gb
```
Comment (astonzhang) [inline chapter_natural-language-processing/sentiment-analysis.md]: Make the same changes as in textcnn.
